config.go

```diff
@@ -9,7 +9,7 @@ import (
 )
 
 // ConfigFile is the name of the config's serialized form.
-const ConfigFile = "faas.yaml"
+const ConfigFile = "function.yaml"
```

Comment:
Wonder whether a shorter file name (`func.yaml`) makes sense ? If we e.g. add `func` as alias this would be a natural naming, too.

Reply:
yeah, this makes sense. I'll modify it